frontend/src/modules/displays/MultiStepWizard.tsx

```diff
@@ -25,6 +30,26 @@ import { commitRequest, commitDraftRequest } from '../../store/controller/action
 import store from '../../store';
 import { DASHBOARD_ROUTE } from '../navigation/constants';
 
+/**
+ * @typedef Data
+ *
+ * Interface for Data
+ *
+ * @prop {string} stateKey Unique identifier for the value
+ * @prop {string} label Label for the value
```

Comment:
How is this label used? What kinds of values should we provide, e.g. a full name of the value such as Heart Rate, or an abbreviation such as HR?

Reply:
Its an abbreviation as of now. But can be modified as requirement